### ISSUE TITLE
datetime variables are shifted by -1 hour when exported using R client

### ISSUE THREAD

**Issue by abhishekdxt** [obiba/opal#3449]
When I export a Opal table containing a datetime variable using the R client (v2.6.0) I see a 
-1 hour shift in values.

For example, 
variable with value in Opal '1970-01-06 15:35:00' when exported via R client shows us up as '1970-01-06 14:35:00'.

I have tried changing my local timezone in R but still get the same result.

**Comment by ymarcon** [comment]
The R vector holding the datetime values is of class `POSIXct` and the display of these values is in the UTC time.

The timezone is an attribute of the POSIXct object, this command will probably return `UTC`:

```
attr(data$dateTime, "tzone")
```

You can change the timezone of this vector using:

```
attr(data$dateTime, "tzone") <- "Europe/London"
```

This will give you the same values as in Opal.

**Comment by ymarcon** [comment]
I just noticed that it's Opal that explicitly builds the POSIXct object with UTC timezone. Maybe it should Opal server's timezone, for consistency?

**Comment by abhishekdxt** [comment]
That is correct. The whole point is that the datetime values shouldn't change upon export via R.

**Comment by ymarcon** [comment]
Well actually the values do not change, it's just the formatting of the display.

**Comment by abhishekdxt** [comment]
Sure but the formatting intrinsically changes the values.
For example, exporting the data as data.frame and then writing the data to a CSV for some other purpose.

**Comment by abhishekdxt** [comment]
Hi, could you possibly revisit this issue? (Thanks)

Now both date and the datetime columns when exported via R appear as epoch timestamp e.g. 1970-01-02 14:00:00 BST in Opal appears as '133200' in R.

**Comment by abhishekdxt** [comment]
Thanks for fixing this issue. Could you also fix a small typo 'commpressed' in the data import UI.

>Data in "delimiter separated values" format are in a text file (.csv or .tsv). The file can be commpressed in a zip file (.zip).

**Comment by abhishekdxt** [comment]
Thank you the date/datetime issue is now successfully resolved in opal-2.14.2. 
Unfortuantely the typo 'commpressed' is still there.
